### PR TITLE
adding another link to learn programming

### PR DESCRIPTION
**Pull request by MohamedRamzy**

### CODE REVIEW

**Comment by MohamedRamzy** [comment]
Please review this, @casey-collab

File: arabic/learn/whattolearn/programming/programming.md [hunk 1]
```diff
@@ -1 +1,2 @@
 ممكن أن تبدأ [بهذا](https://www.udacity.com)
+وهذا ايضا [هنا](https://www.codeschool.com)
```
Comment (casey-collab) [inline arabic/learn/whattolearn/programming/programming.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by MohamedRamzy** [comment]
Please review this again, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :fireworks:

**Pull request merged**